satellite/marketing/marketingweb/server.go

```diff
@@ -83,13 +76,27 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		filepath.Join(s.templateDir, "foModal.html"),
 	)
 
-	home, err := template.New("landingPage").ParseFiles(files...)
+	s.templates, err = template.New("landingPage").ParseFiles(files...)
 	if err != nil {
+		s.log.Error("failed to parse template", zap.Error(err))
+	}
+
+	return s
+}
+
+// ServeHTTP handles index request
+func (s *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	if req.URL.Path != "/" {
+		s.serveNotFound(w, req)
+		return
+	}
+
+	if s.templates == nil {
```

Comment:
There are other places where the templates are parsed.

Reply:
added all templates parsing into a method